csrankings-w.csv

```diff
@@ -72,6 +72,7 @@ Wangmeng Zuo,Harbin Institute of Technology,http://homepage.hit.edu.cn/wangmengz
 Wanli Chang 0001,Hunan University,http://csee.hnu.edu.cn/people/changwanli,OYcT_B4AAAAJ
 Wannida Sae-Tang,KMUTNB,https://tggs.kmutnb.ac.th/asst-prof-dr-wannida-sae-tang,6nlcJK8AAAAJ
 Wanxiang Che,Harbin Institute of Technology,http://ir.hit.edu.cn/~car/english.htm,SVlQ6IEAAAAJ
+Wanyun Cui,SUFE,http://simecv.sufe.edu.cn/page.aspx?id=41,FP7ANisAAAAJ

```

Comment:
Please update to this home page: https://cuiwanyun.github.io/.

Reply:
Sure.